.azure-pipelines/generation-pipeline.yml

```diff
@@ -498,6 +498,60 @@ stages:
              migration: false
              pathToCopy: '/com/Beta/*'
 
+- stage: stage_php_beta_kiota
+  dependsOn:
+    - stage_build_and_publish_kiota
+    - stage_beta_openapi
+  condition: |
+    and
+    (
+      eq(dependencies.stage_build_and_publish_kiota.result, 'Succeeded'),
+      eq(dependencies.stage_beta_openapi.result, 'Succeeded')
+    )
+  jobs:
+    - job: php_beta_kiota
+      steps:
+        - template: generation-templates/language-generation-kiota.yml
+          parameters:
+            language: 'php'
+            version: 'beta'
+            repoName: 'msgraph-beta-sdk-php'
+            branchName: 'kiota/$(betaBranch)'
+            targetClassName: "BaseGraphClient"
+            targetNamespace: 'Microsoft\\Graph\\Beta\\Generated'
+            baseBranchName: 'feat/3.0'
```

Comment:
Could we name the base branches to `feat/kiota-preview` or something clearer for internal stakeholders to find while testing?

Reply:
The Kiota SDK will eventually be the new v2 not a v3 i.e. eventually merged to `feat/2.0.0` and release as a 2.0-preview . I find naming the branch `feat/3.0` confusing/misleading

Reply:
for my education, what's preventing you from using feat/2.0.0 at this point? this is what @andrueastman has used for dotnet (v5 though) https://github.com/microsoftgraph/msgraph-sdk-dotnet/tree/feature/5.0

Reply:
is there any specific reason why php is using a release candidate nomenclature and not an preview ?